### PR TITLE
Move MediaPlayer background to Modifier

### PR DESCRIPTION
**Pull request by yschimke**
#### WHAT

Move MediaPlayer background to Modifier

#### WHY

Drawing the background via a Modifier allows avoiding Offscreen Composition for Marquee Text.

#### Checklist :clipboard:
- [x] Add explicit visibility modifier and explicit return types for public declarations
- [x] Run spotless check
- [x] Run tests
- [x] Update metalava's signature text files

### CODE REVIEW

File: media/ui/src/main/java/com/google/android/horologist/media/ui/components/background/ArtworkColorBackground.kt [hunk 1]
```diff
@@ -43,18 +44,63 @@ import coil.request.ImageRequest
  * Background using a radial gradient extracted from artwork.
  */
 @Composable
+@Deprecated("Prefer background modifier")
```
Comment (luizgrp) [inline media/ui/src/main/java/com/google/android/horologist/media/ui/components/background/ArtworkColorBackground.kt:47]: is the experimental annotation missing for all the components of this package?

File: media/ui/src/test/snapshots/images/com.google.android.horologist.media.ui_MediaPlayerA11yScreenshotTest_mediaPlayerLargeRound.png
Comment (luizgrp) [inline media/ui/src/test/snapshots/images/com.google.android.horologist.media.ui_MediaPlayerA11yScreenshotTest_mediaPlayerLargeRound.png:1]: screenshots seem to be identical, were they updated due to generating them in a non linux machine?

**Review by luizgrp — APPROVED** [review]

File: media/ui/src/main/java/com/google/android/horologist/media/ui/components/background/ArtworkColorBackground.kt (hunk 1 above)
Comment (yschimke) [reply media/ui/src/main/java/com/google/android/horologist/media/ui/components/background/ArtworkColorBackground.kt:47]: I'll add.  I was going to suggest we remove a bunch of these from 0.5.x

Comment (luizgrp) [reply media/ui/src/main/java/com/google/android/horologist/media/ui/components/background/ArtworkColorBackground.kt:49]: Did you mean the ones from https://github.com/google/horologist/pull/1522 ?

I meant the single one that is not deprecated, ExperimentalHorologistApi

**Comment by yschimke** [comment]
Example diffs 
image

image

**Pull request merged**